<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDataTable</class>
 <widget class="QDialog" name="ImportDataTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="ignoreListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>191</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lipidListWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>191</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="featureListWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>191</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>480</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Columns to ignore</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Lipid columns</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Feature columns</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Measurement column</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
